vignettes/EIRprevmatch.Rmd

```diff
@@ -89,85 +173,209 @@ prev <- lapply(
     )
   }
 )
-
-# create dataframe of initial EIR, output EIR, and prev 2-10 results
-EIR_prev <- cbind.data.frame(init_EIR, output_EIR = unlist(EIR), prev = unlist(prev))
+ 
+# Create dataframe of initial EIR, output EIR, and prev 2-10 results
+malSim_P2E <- cbind.data.frame(init_EIR, EIR = unlist(malSim_EIR), prev = unlist(malSim_prev))
 
 ```
 
-Now plot your results! Code is included to compare the results of matching PfPR2-10 to EIR based on `malariaEquilibrium` (blue line) versus matching based on parameterized `malariasimulation` runs (red line). Notice that the generated points do not form a smooth curve. We ran `malariasimulation` using a population of just 5,000. Increasing the population to 10,000 or even 100,000 will generate more accurate estimates, but will take longer to run. 
+### Fit line of best fit relating initial EIR values to *Pf*PR~2-10~
 
-```{r plot}
-# calculate EIR / prev 2-10 relationship from malariaEquilibrium
-eir <- seq(from = 0.1, to = 50, by=.5)
-eq_params <- malariaEquilibrium::load_parameter_set("Jamie_parameters.rds")
+Having run `malariasimulation` simulations for a range of initial EIRs, we can fit a line of best fit through the initial EIR and *Pf*PR~2-10~ data and use it to predict the prevalence for a wider range of initial EIRs (given the set of parameters used). Note that the `newdata` argument is used to generate a series of additional initial EIR and *Pf*PR~2-10~ pairs for plotting.
 
-prev <- vapply( # calculate prevalence between 2:10 for a bunch of EIRs
-  eir,
-  function(eir) {
-    eq <- malariaEquilibrium::human_equilibrium(
-      eir,
-      ft=0,
-      p=eq_params,
-      age=0:100
-    )
-    sum(eq$states[2:10, 'pos_M']) / sum(eq$states[2:10, 'prop'])
-  },
-  numeric(1)
-)
+```{r}
+
+# Fit a line of best fit through malariasimulation initial EIR and prevalence
+malSim_fit <- predict(gam(prev~s(init_EIR, k = 5), data = malSim_P2E), 
```

Comment:
Is there a base R way to predict with gam models? Not sure if it is possible or not

Reply:
I couldn't find one, but have added a paragraph to the introduction to let the user now they can simply uncomment the package load and play with this themselves.